app/controllers/mandatarissen/persoon/mandaten.js

```diff
@@ -49,23 +57,18 @@ export default class MandatarissenPersoonMandatenController extends Controller {
     this.activeOnly = !this.activeOnly;
   }
 
-  checkFracties = task(async (foldedMandatarissen) => {
+  checkFracties = task(async (mandatarissen) => {
     this.canBecomeOnafhankelijk = false;
     this.possibelOnafhankelijkeMandatarissen = [];
-    for (const fold of foldedMandatarissen) {
-      const isActive = await this.mandatarisService.isMandatarisActive(
-        fold.mandataris
-      );
+    for (const mandataris of mandatarissen) {
```

Comment:
Shouldn't we be able to do this as one call?

Reply:
a method `filterNonOnafhankelijkeandatarissen()` could be added to mandataris yes